src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -122,12 +147,16 @@ const std::map<std::string, std::string> GetModuleDataToCapabilitiesMapping() {
   mapping["massageMode"] = "massageModeAvailable";
   mapping["massageCushionFirmness"] = "massageCushionFirmnessAvailable";
   mapping["memory"] = "memoryAvailable";
+
   // audio
   mapping["source"] = "sourceAvailable";
   mapping["keepContext"] = "keepContextAvailable";
   mapping["volume"] = "volumeAvailable";
   mapping["equalizerSettings"] = "equalizerAvailable";
 
+  // light
+  mapping["supportedLights"] = "";
```

Comment:
Why is this an empty string?

Reply:
@JackLivio This parameter already redundant because we are getting the light structure by calling `GetModuleDataToCapabilitiesMapping` function. It was deleted at 2614bb2209111b873c0689156e785d7039bdb39d , please take a look